dask_planner/Cargo.toml

```diff
@@ -12,11 +12,11 @@ rust-version = "1.59"
 tokio = { version = "1.0", features = ["macros", "rt", "rt-multi-thread", "sync", "fs", "parking_lot"] }
 rand = "0.7"
 pyo3 = { version = "0.16", features = ["extension-module", "abi3", "abi3-py38"] }
-arrow = { version = "18.0.0", features = ["prettyprint"] }
```

Comment:
Waiting for changes to be merged upstream and will update this reference back to Apache